README.md

```diff
@@ -10,7 +10,11 @@
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 [![npm](https://img.shields.io/npm/dm/level.svg)](https://www.npmjs.com/package/level)
 
-A convenience package that bundles the current release of [`levelup`][levelup] and [`leveldown`][leveldown] and exposes `levelup` on its export.
+A convenience package that:
+
+* exports a function that returns a [`levelup instance`](https://github.com/level/levelup#ctor) when invoked
+* bundles the current release of [`levelup`][levelup] and [`leveldown`][leveldown]
+* leverages encodings using [`encoding-down`][encoding-down]
 
```

Comment:
@vweevers I think this is a bit clearer at least. Let me know if you want to tweak it.

Reply:
This is much better already. We can tweak language later if needed; let's first add documentation to `encoding-down` so we have all the parts covered.

Reply:
@vweevers WDYT is missing from `encoding-down` docs?

Reply:
@ralphtheninja https://github.com/Level/encoding-down/issues/15